layers/meta-resin-edison/recipes-bsp/u-boot/files/target_env/blankcdc.env

```diff
@@ -7,4 +7,4 @@ bootargs_target=multi-user
 bootargs_ethconfig=cdc
 dfu_to_sec=3
 do_probe_dfu=echo "Probing DFU..." ; run do_dfu_alt_info_mmc ; sleep 2 ; saveenv ; dfu 0 mmc 0 ${dfu_to_sec}
-boot_target_cmd=run do_flash_os;run do_probe_dfu;run do_compute_target;run mmc-bootargs;run load_kernel;zboot ${loadaddr}
+boot_target_cmd=setenv resin_kernel_load_addr ${loadaddr}; run resin_set_kernel_root; run do_flash_os;run do_probe_dfu;run do_compute_target;run mmc-bootargs;run load_kernel;zboot ${loadaddr}
```

Comment:
You need to have a check here to see if resin_set_kernel_root is defined and only then run it. Otherwise you will get errors at the edison flashing stage with the blankcdc.env you have here because at that time you will be running the old uboot on the edison which does not yet have defined resin_set_kernel_root